src/lib/rest/rest.cpp

```diff
@@ -1002,7 +996,6 @@ static int connectionTreat
   }
 
 
-  //
```

Comment:
Hey, I want this line ... :-)
I use to make 'important' comments about the flow in the code like this:

    //
    // Now we do this ...
    //